app/javascript/react/screens/App/Plan/PlanConstants.js

```diff
@@ -29,7 +30,8 @@ export const STATUS_MESSAGE_KEYS = {
   VALIDATING: 'Validating',
   VM_MIGRATIONS_COMPLETED: 'VM Transformations completed',
   VM_MIGRATED: 'Virtual machine migrated',
-  VM_MIGRATIONS_FAILED: 'VM Transformations failed'
+  VM_MIGRATIONS_FAILED: 'VM Transformations failed',
+  CANCELLED: 'VM cancelled'
```

Comment:
I have removed this constant for now, since we don't know yet what string the state machine actually uses for a cancelled VM.

This will become more clear after some appliance testing and once Fabien's cancellation implementation is complete